Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu/mysql:8.0-20.04_beta
+
+ENV CONFIG_VERSION="0.8.22-1"
```

Comment:
This should probably be an ARG, not ENV - otherwise it'll be present at runtime too (unless that's what you intended?)

Reply:
changed to arg in [69ec1f1](https://github.com/canonical/mysql-container/pull/1/commits/69ec1f179f718412815e44d05e7e846f96774445)